examples/ssh/opkssh/cli.go

```diff
@@ -0,0 +1,319 @@
+package main
```

Comment:
every function in this file should be privatized

Reply:
done